docs/components/Header.tsx

```diff
@@ -1,26 +1,26 @@
-import { MenuIcon } from '@heroicons/react/outline'
+import { useNavData } from '@/components/NavDataContext'
+import { ArrowLeftIcon, MenuIcon } from '@heroicons/react/outline'
 import clsx from 'clsx'
 import { useRouter } from 'next/dist/client/router'
 import Link from 'next/link'
-import React, { useEffect } from 'react'
-import { useState } from 'react'
-
+import React, { MouseEvent, useEffect, useState } from 'react'
 import Button from '../components/ui/Button'
+import { Nav } from './Nav'
 
 export default function Header() {
   const [mobileMenu, setMobileMenu] = useState(false)
   const router = useRouter()
   const GithubStarsCounter = () => {
     const repoUrl = `https://api.github.com/repos/nhost/nhost`
-    const [count, setCount] = useState(null);
-    const format = n => n > 1000 ? `${(n / 1000).toFixed(1)}k` : n;
+    const [count, setCount] = useState(null)
+    const format = (n: number) => (n > 1000 ? `${(n / 1000).toFixed(1)}k` : n)
```

Comment:
how could the previous code even work without types here? is it something with the tsconfig we should update to not have to patch these things in the future?